kbase-extension/scss/partials/_narrative.scss

```diff
@@ -126,24 +126,31 @@
   table-layout: fixed;
 }
 
-.kb-staging-table-header__checkbox {
-  width: 2%;
-}
+.kb-staging-table .kb-staging-table-header {
 
-.kb-staging-table-header__file {
-  width: 4%;
-}
+  .kb-staging-table-header__checkbox {
+    width: 20px;
+    padding-right: 0;
```

Comment:
Does this need `text-align: center` adding?

Reply:
no i think we should be ok without that here - unless you're seeing a visual bug!